src/data/fin.lean

```diff
@@ -1238,6 +1241,9 @@ variables {α : fin (n+1) → Type u} (x : α (last n)) (q : Πi, α i) (p : Π(
 def init (q : Πi, α i) (i : fin n) : α i.cast_succ :=
 q i.cast_succ
 
+lemma init_lambda {n : ℕ} {α : fin (n+1) → Type*} {q : Π i, α i} :
```

Comment:
Is `_lambda` a common pattern in mathlib for this sort of thing? I can't think of a better suggestion though, maybe `init_def`?

Reply:
I agree `init_def` is a better name.